### PR TITLE
Parsed more cards + minor spelling fix

### PR DESCRIPTION
**Pull request by turt2live**

### CODE REVIEW

**Comment by jkcclemens** [comment]
Make sure there is ending punctuation at the end of black cards, and ensure there is NO ending punctuation at the end of white cards.

File: othercardpacks/Alternia.cards [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+What is your OT3? (Draw 2, play 3.).
+What makes your kokoro go "doki doki"?
+What's in the box, Jack?
+When a bucket is unavailable, trolls with use _.
+When Dave received _ from his Bro for his 9th birthday, be felt a little warm inside.
+Whenever I see _ on MSPARP, I disconnect immediately.
+where doing it man. where MAKING _ HAPEN!
+Your name is JOHN EGBERT and boy do you love _!
+___WHITE___
+â€¦ . .-. . -. .. - -.â€”  (Serenity)
```
Comment (jkcclemens) [inline othercardpacks/Alternia.cards]: I feel like this is an encoding issue.

File: othercardpacks/Alternia.cards [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+The secret way to achieve God Tier is to die on top of _.
+The thing that made Kankri break his vow of celibacy was _.
+Turns out, pre-entry prototyping with _ was not the best idea.
+Vriska killed Spidermom with _.
+Vriska roleplays _ with Terezi as _.
+Vriska's greatest regret is _.
+Wear  _. Be _.
+What did Jake get Dirk for his birthday?
+What is the worst thing that Terezi ever licked?
+What is your OT3? (Draw 2, play 3.).
```
Comment (jkcclemens) [inline othercardpacks/Alternia.cards]: So, we don't need instructions in the card.

File: othercardpacks/Alternia.cards [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+What makes your kokoro go "doki doki"?
+What's in the box, Jack?
+When a bucket is unavailable, trolls with use _.
+When Dave received _ from his Bro for his 9th birthday, be felt a little warm inside.
+Whenever I see _ on MSPARP, I disconnect immediately.
+where doing it man. where MAKING _ HAPEN!
+Your name is JOHN EGBERT and boy do you love _!
+___WHITE___
+... . .-. . -. .. - -.-"  (Serenity)
```
Comment (jkcclemens) [inline othercardpacks/Alternia.cards]: Missing the start quote now?

File: othercardpacks/Alternia.cards [hunk 4]
```diff
@@ -0,0 +1,432 @@
...
+Rose and Kanaya snuggling
+Rose Lalonde
+Rose telling John she's a lesbian and they will never be together
+Rose's review of "My Immorta"
+Roxy Lalonde
+Rufio
+Rufioh Nitram
+Sacred leggings
+SBAHJ hentai doujinshi
+SchrÃ¶dinger's Nepeta
```
Comment (jkcclemens) [inline othercardpacks/Alternia.cards]: Encoding

File: othercardpacks/Alternia.cards (hunk 3 above)
Comment (turt2live) [inline othercardpacks/Alternia.cards]: Nope. Raw line: 
`{"id":2074,"cardType":"A","text":"… . .-. . -. .. - -.—  (Serenity)","numAnswers":0,"expansion":"Alternia"},`

File: othercardpacks/CAHgrognards.cards [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+Random chargen
+RPG.net
+Dice inserted somewhere painful
+FATAL
+Ron Edwards' brain damage
+Boob plate armor
+Gamer chicks
+GNS theory
+Drizzt
+The entire Palladium Books&reg; Megaverse&trade;
```
Comment (jkcclemens) [inline othercardpacks/CAHgrognards.cards]: HTML entities

**Comment by jkcclemens** [comment]
Alphabetize cards.

File: othercardpacks/Alternia.cards (hunk 3 above)
Comment (jkcclemens) [inline othercardpacks/Alternia.cards]: Extra end quote, then?

**Pull request merged**